src/test/storage/single_cxlumn_index_test.cpp

```diff
@@ -19,52 +19,53 @@
 namespace opossum {
 
 template <typename DerivedIndex>
-class SingleColumnIndexTest : public BaseTest {
+class SingleSegmentIndexTest : public BaseTest {
```

Comment:
@mrks I think there is an inconsistency here.
The file was renamed to `single_cxlumn_index_test`, but the test name to `SingleSegmentIndexTest`.
For `multi_cxlumn_index_test` the test name is `MultiCxlumnIndexTest`.
I think we should be consistent, whichever names we choose.
I personally think that we should stick to column index because people might get more confused when they read segment index, but I can see it either way.

Reply:
As indexes are on a per-chunk level, I would clearly consider them to be SingleSegment and MultiSegment and have renamed the test.